test/unit/Cover/buyCover.js

```diff
@@ -88,6 +88,49 @@ describe('buyCover', function () {
     });
   });
 
+  it('should purchase new cover with fixed price using 1 staking pool', async function () {
+    const { cover } = this;
+
+    const {
+      members: [coverBuyer],
+    } = this.accounts;
+
+    const { amount, targetPriceRatio, coverAsset, period, expectedPremium, coverId } = buyCoverFixture;
```

Comment:
remove coverId and read it after the buyCover tx with `const coverId = await cover.coverData.length`